tests/compile-fail/ptr_rem.rs

```diff
@@ -0,0 +1,5 @@
+fn main() {
+    let val = 13usize;
+    let addr = &val as *const _ as usize;
+    let _ = addr % 2; //~ ERROR access part of a pointer value as raw bytes
```

Comment:
Why is this problematic? Shouldn't this give us zeroes since `val` is aligned to 4 or 8 bytes so the the remainder is always zero?

Reply:
Hm I think you are right, I got this the wrong way around.